operations/transaction-service/.choreo/component.yaml

```
@@ -0,0 +1,29 @@
+# Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com/) All Rights Reserved.
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+schemaVersion: "1.1"
+endpoints:
+    - name: wso2-token-transactional-api-d09
+      displayName: wso2-token-transactional-api
+      service:
+        basePath: /
+        port: 4000
+      type: REST
+      networkVisibilities:
+        - Project
+        - Public
+      schemaFilePath: openapi.yaml
```

(minor) Omit extra new lines, and add a new line at the EoF (for propper Unix file appending).
```suggestion
schemaVersion: "1.1"
endpoints:
    - name: wso2-token-transactional-api-d09
      displayName: wso2-token-transactional-api
      service:
        basePath: /
        port: 4000
      type: REST
      networkVisibilities:
        - Project
        - Public
      schemaFilePath: openapi.yaml

```